packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -977,6 +977,12 @@ export interface IRetriableFailureResult {
     readonly retryAfterSeconds?: number;
 }
 
+// @alpha
+export interface IRetriableSummaryError extends Error {
```

Comment:
My only question here is whether you want to introduce a new errorType.  I think not, you're just trying to introduce type safety internally, whatever the underlying error / errorType is.

Reply:
Ok but later you're using `wrapError` which will occlude the original errorType anyway. So, it's something to consider.

Where, if anywhere, would these errors escape to the Host?

Reply:
I could introduce a new error type which actually exists in `RetriableSummaryError` class. However, the other error types are all internal so I wasn't sure if I should make one alpha.

Reply:
This will be used internally in Fluid, right and will not be exposed to the hosts, right?

Reply:
Should be ok to add an errorType even if it is internal because there should be some prop which would be a differentiating factor if someone lets say wants to check that this error is indeed a summarizer related error. Without that it would be difficult to say it without confidence.

Reply:
It is alpha not internal.

Reply:
This PR / discussion is getting big. My focus right now is getting this change in. So, I removed the new error type and reverted to the way it was before. I will consider making this type safe in a follow up PR.